internal/pkg/cli/pipeline_status.go

```diff
@@ -0,0 +1,80 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const (
+	pipelineStatusAppNamePrompt     = "Which application is the pipeline in?"
+	pipelineStatusAppNameHelpPrompt = "An application is a collection of related services."
+	pipelineStatusNamePrompt        = "Which pipeline's status would you like to show?"
+	pipelineStatusNameHelpPrompt    = "Displays the status of each of a pipeline's stages."
+)
+
+type pipelineStatusVars struct {
+	*GlobalOpts
+	shouldOutputJSON bool
+	appName       string
+	pipelineName  string
+}
+
+type pipelineStatusOpts struct {
+	pipelineStatusVars
+}
+
+func newPipelineStatusOpts(vars pipelineStatusVars) (*pipelineStatusOpts, error) {
+	return &pipelineStatusOpts{
+		pipelineStatusVars: vars,
+	}, nil
+}
+
+// Validate returns an error if the values provided by the user are invalid.
+func (o *pipelineStatusOpts) Validate() error {
+	return nil
+}
+
+// Ask prompts for fields that are required but not passed in.
+func (o *pipelineStatusOpts) Ask() error {
+	return nil
+}
+
+// Execute displays the status of the pipeline.
+func (o *pipelineStatusOpts) Execute() error {
+	return nil
+}
+
+// BuildPipelineStatusCmd builds the command for showing the status of a deployed pipeline.
+func BuildPipelineStatusCmd() *cobra.Command {
+	vars := pipelineStatusVars{
+		GlobalOpts: NewGlobalOpts(),
+	}
+	cmd := &cobra.Command{
+		Hidden: true, // TODO: remove when ready for production!
+		Use:    "status",
+		Short:  "Shows the status of a deployed pipeline.",
+		Long:   "Shows the status of each deployed pipeline's stages.",
```

Comment:
I think we had talked about this being "Show the status of your application's pipeline." but open to this one. Thoughts @efekarakus @kohidave ?